manifests/t/timedomain/acestudio/0.7.11.5/timedomain.acestudio.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: timedomain.acestudio
+PackageVersion: 0.7.11.5

```

```suggestion
PackageVersion: 1.0.2
```
This must match Control Panel to prevent upgrade loops.